JavaScript/JavaScriptSDK/JavaScriptSDK.csproj

```diff
@@ -56,6 +56,15 @@
     <TypeScriptRemoveComments>false</TypeScriptRemoveComments>
     <TypeScriptSourceMap>true</TypeScriptSourceMap>
     <TypeScriptModuleKind>AMD</TypeScriptModuleKind>
+    <TypeScriptJSXEmit>None</TypeScriptJSXEmit>
+    <TypeScriptCompileOnSaveEnabled>True</TypeScriptCompileOnSaveEnabled>
+    <TypeScriptNoImplicitAny>False</TypeScriptNoImplicitAny>
+    <TypeScriptOutFile>min\$(OutFileBaseName).js</TypeScriptOutFile>
+    <TypeScriptOutDir />
+    <TypeScriptNoEmitOnError>True</TypeScriptNoEmitOnError>
+    <TypeScriptMapRoot>
+    </TypeScriptMapRoot>
+    <TypeScriptSourceRoot>https://raw.githubusercontent.com/Microsoft/ApplicationInsights-JS/master/JavaScript</TypeScriptSourceRoot>
```

Comment:
I saw it but for some reason, it didn't work for me when I tried it. I guess it was the issue with the path capitalization because now it is working without any issues.